### PR TITLE
Added Bedrock Block Exporter

### PR DESCRIPTION
**Pull request by Brennian**
Created new plugin for creating new Blocks on Bedrock Edition

### CODE REVIEW

File: plugins/arcaniax_block_exporter.js [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+		title: 'Bedrock Block Exporter',
+		icon: 'icon-format_block',
+		author: 'Arcaniax',
+		description: 'Helps making new Bedrock blocks (requires experimental mode)',
+		version: '0.1.1',
+		variant: 'both',
+
+		onload() {
+
+			Blockbench.addMenuEntry('Bedrock Block Exporter', 'icon-format_block', function() {
```
Comment (JannisX11) [inline plugins/arcaniax_block_exporter.js:437]: Please use Actions here. You can find an example in many newer plugins, e. g. "Plaster".
`addMenuEntry` is a deprecated method from the old days of Blockbench.

File: plugins/arcaniax_block_exporter.js [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+	}
+	
+	
+	function getResourceManifest(formData){
+		var name = formData.name;
+		var uuid1 = generateUUID();
+		var uuid2 = generateUUID();
+		var json =
+`{
+    "format_version": 2,
```
Comment (JannisX11) [inline plugins/arcaniax_block_exporter.js:260]: Can't help but notice that the JSON files all use a different type of indentation. Maybe use a consistent method of indentation, like only tabs.
You could also use the built-in function `compileJSON()` to generate the JSON strings, then you won't need to worry about indentation and you can also write the JSON code directly in code, without having to deal with template strings.

**Comment by JannisX11** [comment]
Also, please add the plugin to the `plugins.json` file.

File: plugins/arcaniax_block_exporter.js (hunk 1 above)
Comment (Brennian) [reply plugins/arcaniax_block_exporter.js]: Changed to the newer method.

**Pull request merged**